src/Core/tests/Benchmarks/Benchmarks/StyleBenchmarker.cs

```diff
@@ -0,0 +1,52 @@
+﻿using BenchmarkDotNet.Attributes;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.Benchmarks;
+
+[MemoryDiagnoser]
+public class StyleBenchmarker
+{
+	[Benchmark]
+	public void MergedStyle()
+	{
+		var buttonStyle = new Style(typeof(Button))
+		{
+			Setters = {
+				new Setter { Property = Button.TextColorProperty, Value = Colors.Pink },
+			},
+			Class = "pink",
+			ApplyToDerivedTypes = true,
+		};
+		var labelStyle = new Style(typeof(Label))
+		{
+			Setters = {
+				new Setter { Property = Button.BackgroundColorProperty, Value = Colors.Pink },
+			},
+			Class = "pink",
+			ApplyToDerivedTypes = false,
+		};
```

Comment:
What if the styles were created as member variables in the Benchmark class? It might narrow the work happening in the `MergedStyle()` method and get us a more accurate benchmark? It will run that method many times.

If you could get the benchmark down to just merging the styles and creating the fewest objects, that would be good.